tools/doc/html.js

```diff
@@ -137,6 +143,41 @@ function render(opts, cb) {
   });
 }
 
+function altDocs(filename) {
+  let html = '';
+
+  if (!docCreated) {
+    console.error('Failed to add alternative version links');
+    return html;
+  }
+
+  function lte(v) {
+    if (docCreated[1] > v[0])
+      return false;
+    if (docCreated[1] < v[0])
+      return true;
+    return docCreated[2] <= v.substr(2, 2);
+  }
+
+  const host = 'https://nodejs.org';
+  const href = (v) => `${host}/docs/latest-v${v}/api/${filename}.html`;
+  const a = (v) => `<a href="${href(v)}">v${v}</a>`;
+  const as = (vs) => vs.filter(lte).map(a).join(' / ');
+
+  const lts = as(['0.12.x', '4.x', '6.x']);
+  if (lts.length)
+    html += '<b>LTS:</b> ' + lts;
+
+  const unsupported = as(['0.10.x', '5.x', '7.x']);
```

Comment:
v7.x is still a supported version, so calling it `Unsupported` would be confusing IMO